client/src/application/containers/utils/navbar/index.tsx

```diff
@@ -112,7 +129,8 @@ class Navbar extends React.Component<IProps, IState> {
 
         const logo = <BPNavbar.Heading>
             <span dangerouslySetInnerHTML={{__html: arkhnLogoWhite}} />
-            <h2>PYROG</h2>
+            {this.renderRedirect()}
+            <h2 onClick={this.setRedirect}>PYROG</h2>
```

Comment:
Just a few comments:
* Rather than ```
onClick={this.setRedirect}
```
try and use this:
```
onClick={this.props.history.push('/')
```
It will allow this redirection to be stored in React redirection history.
* Try to use `onClick` directly in `BPNavbar.Heading`